src/main/webapp/js/actions/app-settings-actions.js

```diff
@@ -0,0 +1,22 @@
+import apiClient from '../utils/api-client';
+import { FETCH_APP_SETTINGS, UPDATE_APP_SETTINGS } from './types';
+import { APP_SETTINGS_ENTITY } from '../constants/entity-types';
+
+const BASE_URL = '/api';
+
+export const fetchAppSettings = () => {
+  const request = apiClient.get(`${BASE_URL}/${APP_SETTINGS_ENTITY}`);
```

Comment:
I don't think you will need to the use entity const here, just change the base url